src/app/components/elements/Userpic.jsx

```diff
@@ -3,26 +3,41 @@ import { connect } from 'react-redux';
 import shouldComponentUpdate from 'app/utils/shouldComponentUpdate';
 import { imageProxy } from 'app/utils/ProxifyUrl';
 
+export const SIZE_SMALL = 'small';
+export const SIZE_MED = 'medium';
+export const SIZE_LARGE = 'large';
+
+const sizeList = [SIZE_SMALL, SIZE_MED, SIZE_LARGE]
+
+export const avatarSize = {
```

Comment:
what's this for? it's not referenced anywhere

Reply:
It's for `<Userpic size={avatarSize.small} />`

Reply:
wanna roll that usage into this branch so we can start using it & save some kb/pageload right off the bat?

Reply:
Bring it on in.

Reply:
https://github.com/steemit/condenser/pull/1909/commits/4f9b508423f9f79e8fe97b74574ce44b8185f042 < - brought it in